src/ui/public/react_components.js

```diff
@@ -0,0 +1,15 @@
+import 'ngreact';
```

Comment:
Question: Should we collect all ui framework components in here, or try to keep this file updated with only those that are currently being used?